_posts/2019-08-08-hacking-for-cruise-control.md

```diff
@@ -0,0 +1,227 @@
+---
+layout: post
+title:  "Hacking Strimzi for Cruise Control"
+date: 2019-08-08
+author: kyle_liberti
+---
+
+Cruise Control is a higly customizable Kafka partition balancing and monitoring software run in production at Linkedin. Cruise Controls’s component pluggability makes it flexible to customize for different environments. This makes it possible to extend Cruise Control to work with Strimzi in an Kubernetes environment with a reasonable amount of effort. This post will walk through how to get Cruise Control working with a Stimzi deployed Kafka cluster.
+
+# Prequisite Outline
```

Comment:
you have to use `<!--more-->` before this title in order to separate the first part that will be in the posts list as quick introduction.

Reply:
```suggestion
<!--more-->

# Prequisite Outline
```